frontend/src/AlertConfiguration/AlertConfigurationComponents/ControlPanel.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import ConfigForm from './ConfigForm'
+
+const useStyles = makeStyles({
+  controlPanel: {
+    backgroundColor: 'white',
+    padding: 50,
+    width: '100wv'
+  },
+  configForm: {
+    padding: 25,
+  },
+});
+
+/** 
+ * ControlPanle is a control panel for creating new configurations and deleting old configurations.
+ * It takes as input addConfig (a function for creating new configurations).
+ */
+export default function ControlPanel({ addConfig }) {
+  const classes = useStyles();
+
+  const [displayConfigForm, setDisplayConfigForm] = React.useState(false);
```

Comment:
ooc, what's the difference between `useState` and `React.useState`?

Reply:
There isn't a difference. I just altered my import statements so that I only use useState instead of React.useState of consistency